collector/log/src/main/cfg/org.apache.karaf.decanter.collector.log.cfg

```diff
@@ -33,3 +33,6 @@ ignored.categories=org.apache.karaf.decanter.appender.*
 #fields.rename.level=severity
 #fields.rename.renderedMessage=logMessage
 #fields.remove.timestamp=true
+
+# disable location information
+location.disabled=false # default
```

Comment:
So you are proposing to disable this globally ? Maybe it could be useful for some log events and not for others no ?